WalkingChallenge/DashboardV2/DashboardDataSource.swift

```diff
@@ -47,9 +47,7 @@ class DashboardDataSource: TableViewDataSource {
         self?.configure()
         completion()
       }
-    }
-
-    onBackground { [weak self] in
+      
       Facebook.getRealName(for: "me") { (name) in
         self?.name = name ?? "Could not load"
```

Comment:
Should we have a different placeholder or something else so that it doesn't look like we are representing that as a name?